be/src/storage/compaction_manager.h

```diff
@@ -133,6 +141,10 @@ class CompactionManager {
     int32_t _max_dispatch_count = 0;
 
     int32_t _max_task_num = 0;
+    int64_t _base_compaction_concurrency = 0;
+    int64_t _cumulative_compaction_concurrency = 0;
+    double _last_score = 0;
```

Comment:
I can't get the idea of last_score. What's the benefit of the previous last score?

Reply:
Record the change of the latest compaction score, the candidate may be empty and cannot be replaced by the maximum score of the candidate.